swag-chart/templates/microservice_config.yml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: env-vars
+data:
+  EUREKA_CLIENT_SERVICEURL_DEFAULTZONE: http://{{ .Values.eureka.serviceName }}:{{ .Values.eureka.containerPort }}/eureka
+  eureka.client.service-url.defaultZone: http://{{ .Values.eureka.serviceName }}:{{ .Values.eureka.containerPort }}/eureka

```

Comment:
we need both of these?

Reply:
Yes. Gateway service uses the all caps one.

Reply:
if one of them's used by just one of the deployments like that, add a comment to that effect here